cmd/promtool/main.go

```diff
@@ -715,9 +720,23 @@ func CheckRules(ls lintConfig, files ...string) int {
 }
 
 func checkRules(filename string, lintSettings lintConfig) (int, []error) {
```

Comment:
Do you think we can take a slice if files here, and check if len == 0 for stdin?